grpc/src/main/java/com/linecorp/armeria/server/grpc/GrpcServiceBuilder.java

```diff
@@ -177,8 +180,16 @@ public GrpcServiceBuilder enableUnframedRequests(boolean enableUnframedRequests)
      * {@link com.linecorp.armeria.server.ServerBuilder#serviceUnder(String, Service)}.
      */
     public Service<HttpRequest, HttpResponse> build() {
+        HandlerRegistry handlerRegistry = registryBuilder.build();
         GrpcService grpcService = new GrpcService(
-                registryBuilder.build(),
+                handlerRegistry,
+                handlerRegistry
+                      .methods()
+                      .keySet()
+                      .stream()
+                      .sorted(String::compareTo)
```

Comment:
`.sorted(String::compareTo)` could be replaced with `sorted()` given that `String` is `Comparable` already.